frontend/src/features/layers/base/BaseLayerRow.tsx

```diff
@@ -0,0 +1,38 @@
+import { Radio } from 'rsuite'
+import styled from 'styled-components'
+
+import { COLORS } from '../../../constants/constants'
+import { BaseLayers } from '../../../domain/entities/layers/constants'
+
+type BaseLayerRowProps = {
+  layer: string
+}
+export function BaseLayerRow({ layer }: BaseLayerRowProps) {
+  return (
+    <Row className="base-layers-selection">
+      <Radio value={layer}>{BaseLayers[layer].text}</Radio>
+    </Row>
+  )
+}
+
+const Row = styled.li`
+  padding: 6px 5px 5px 20px;
+  margin: 0;
+  text-align: left;
+  list-style-type: none;
+  width: 100%;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden !important;
+  cursor: pointer;
+  background: ${COLORS.background};
+  color: ${COLORS.gunMetal};
+  border-bottom: 1px solid ${COLORS.lightGray};
+  display: block;
+  line-height: 18px;
+  user-select: none;
+
+  :hover {
+    background: ${COLORS.shadowBlueLittleOpacity};
```

Comment:
`shadowBlueLittleOpacity` is not part of official colors anymore I think.

Reply:
Indeed, but we need a global rework of the map layers box : regulations, administratives and base layers. As all colors but also icons, margins, etc. changed.
I propose to share the components of Env in the UI and reuse them in Fish.